src/model/user-report.model.ts

```diff
@@ -0,0 +1,26 @@
+import { prop, mongoose } from '@typegoose/typegoose';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UserReport {
+    _id?: mongoose.Types.ObjectId;
+
+    get id() {
+        return this._id;
+    }
+
+    @ApiProperty()
+    @prop({ required: true })
+    title: string;
+
+    @ApiProperty()
+    @prop({ required: true })
+    description: string;
+
+    @ApiProperty()
+    @prop({ required: true })
+    reporterUserId: string;
```

Comment:
I think `reporter` is fine. Also, this should reference to `User` model (see: https://typegoose.github.io/typegoose/guides/advanced/reference-other-classes/)